specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/management.json

```diff
@@ -2037,20 +2035,7 @@
       "description": "Tenant level settings request parameter.",
       "schema": {
         "$ref": "#/definitions/CreateOrUpdateSettingsRequest"
-      },
-      "x-ms-parameter-location": "client"
-    },
-    "OperationResultIdParameter": {
```

Comment:
This parameter is removed because there is no reference of this within the scope of this RP.